clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperationSpec.groovy

```diff
@@ -1760,4 +1760,131 @@ class CreateServerGroupAtomicOperationSpec extends CommonAtomicOperation {
     ex.message == "ECS account settings for account Test do not allow tagging as `serviceLongArnFormat` and `taskLongArnFormat` are not enabled."
   }
 
+  def 'should return valid execution task def arn for non-china and non-gov-cloud partition'() {
```

Comment:
I think you mean "task **execution role** arn" (same goes for other added test cases)

Reply:
Ohh, yes.